test/NuGet.Core.Tests/NuGet.Packaging.Test/SigningTests/SigningCertificateTests.cs

```diff
@@ -77,12 +77,14 @@ public void Read_WithMultipleEssCertIds_ReturnsSigningCertificate()
 
         private static BcEssCertId CreateBcEssCertId(string text)
         {
+#pragma warning disable CS0618 // Type or member is obsolete
             using (var hashAlgorithm = CryptoHashUtility.GetSha1HashProvider())
+#pragma warning restore CS0618 // Type or member is obsolete
```

Comment:
The obsolete message says to use `GetSha1HashProvider`. Perhaps this test code should use that instead.